src/components/LearningTargetForm/LearningTargetForm.js

```diff
@@ -65,7 +62,7 @@ class LearningTargetForm extends React.Component {
               required
               />
             <div>
-              <button type='submit' className='button green-button'>Start Session</button>
+              <button type='submit'>Start Session</button>
```

Comment:
The 'className='button green-button' needs to be there for our CSS

Reply:
Sorry I don't know why those did that.